README.md

```diff
@@ -1 +1,72 @@
-# angular-transclusion-readme
\ No newline at end of file
+# Transclusion
+
+## Overview
+
+We've already used the `transclude` property, but we don't quite know what it is yet. Let's see how we can use the `transclude` property to customize content inside of our directives. 
+
+## Objectives
+
+- Describe transclusion
+- Describe the "transclude" property
+- Use transclusion to inject DOM nodes into our Directive
+- Use ng-transclude
+
+## Transclusion
+
+Say we want to be able to customize the content inside of our directives, like so:
+
+```html
+<our-directive>
+	Content here! We want to put custom content in our directive!
+</our-directive>
+```
+
+Up until now, any content *inside* the directive would be replaced by our template. However, we can change that and actually put the content into our template, and also control where we want to put it - awesome!
+
+To do this, we add a property named `transclude` to our directive. We will set this to `true` for now.
```

Comment:
what does that do? Does that allow us to use the `ng-transclude` directive inside of our template?